api/api.router.js

```diff
@@ -6,6 +6,6 @@ const controllers = require("./api.controllers");
 router.use("/cases", casesRouter);
 
 // /api routes to the api controller and sends back a basic status message to indicate API is up
-router.route("/").get(controllers.apiRoot);
+router.route("/deaths").get(controllers.apiRoot);
```

Comment:
This route is just for /api/. If you hit localhost:3300/api, you'll get a message confirming the server is up and running so you can actually leave this one as just "/".

To add your route, you'll want to add:

```suggestion
const racingRouter = require("../resources/visualizations/racingChart/racing.router");
router.use("/deaths", racingRouter);

router.route("/").get(controllers.apiRoot);
```